rotkehlchen/chain/substrate/typing.py

```diff
@@ -0,0 +1,108 @@
+from enum import Enum
+from typing import Any, Dict, List, NamedTuple, NewType, Tuple, Type, Union
+
+from rotkehlchen.assets.asset import Asset
+from rotkehlchen.fval import FVal
+
+ChainId = NewType('ChainId', str)
+BlockNumber = NewType('BlockNumber', int)
+
+
+class KusamaNodeName(Enum):
+    """
+    TODO: this requires to be extended with OWN_NODE.
+    """
+    PARITY = 0
+    WEB3_FOUNDATION = 1
+
+    def __str__(self) -> str:
+        if self == KusamaNodeName.PARITY:
+            return 'parity'
+        if self == KusamaNodeName.WEB3_FOUNDATION:
+            return 'web3 foundation'
+        raise AssertionError(f'Unexpected KusamaNodeName: {self}')
+
+    def endpoint(self) -> str:
+        if self == KusamaNodeName.PARITY:
+            return 'wss://kusama-rpc.polkadot.io/'
+        if self == KusamaNodeName.WEB3_FOUNDATION:
+            return 'wss://cc3-5.kusama.network/'
+        raise AssertionError(f'Unexpected KusamaNodeName: {self}')
+
+    def is_open(self) -> bool:
```

Comment:
What is this function? Seems like it's a function always returning True?

Reply:
Just an old idea I didn't removed.